### DISCUSSION TITLE
RSS update feed

### DISCUSSION THREAD

**Discussion by AlexanderRitter02** [d2phap/ImageGlass#889] [Q&A] [answered]
I'm using RSS feeds to get notified about software updates and would like to do it with ImageGlass as well.

So I was wondering whether https://imageglass.org/news provides feed funtionality?
I searched for the term "Feed" and "RSS" on the site but didn't find anything.

Does anybody know more and/or have alternatives to getting notified about updates?

**Comment by d2phap** [comment, marked as answer]
Hi @AlexanderRitter02 

You can subscribe to Github feed:
https://github.com/d2phap/imageglass/releases.atom

For ImageGlass website, there is no RSS feed link, but you can follow this link to get the latest update:
- XML: https://imageglass.org/checkforupdate
- JSON: https://imageglass.org/checkforupdate?json

**Comment by AlexanderRitter02** [comment]
Thank you that's great. I wasn't aware of the Github feeds, that's something useful I learned today. 👍
